### PR TITLE
Cvp 1793 multiple ocp

### PR DESCRIPTION
**Pull request by J0zi**
Start different clusters for testing.

@droslean, please have a look

### CODE REVIEW

**Review by mvalarh — APPROVED** [review]

File: ci-operator/config/operator-framework/community-operators/operator-framework-community-operators-master__4.6.yaml [hunk 1]
```diff
@@ -22,6 +22,7 @@ tests:
     test:
     - as: deploy-operator
       commands: |
+        export OCP_CLUSTER_VERSION=4.6
```
Comment (droslean) [inline ci-operator/config/operator-framework/community-operators/operator-framework-community-operators-master__4.6.yaml:25]: I am not sure why this is needed here.

**Review by droslean — APPROVED** [review]

**Pull request merged**